[0.012s] Invoking command in '/home/temur/INS-GPS-ROS2/build/ht_nav_variables': CMAKE_PREFIX_PATH=/opt/ros/foxy /usr/bin/cmake /home/temur/INS-GPS-ROS2/src/ht_nav_variables -DCMAKE_INSTALL_PREFIX=/home/temur/INS-GPS-ROS2/install/ht_nav_variables
[0.031s] -- Found ament_cmake: 0.9.8 (/opt/ros/foxy/share/ament_cmake/cmake)
[0.055s] -- Using PYTHON_EXECUTABLE: /usr/bin/python3
[0.215s] -- Found rosidl_default_generators: 1.0.1 (/opt/ros/foxy/share/rosidl_default_generators/cmake)
[0.218s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.220s] -- Found rosidl_adapter: 1.2.1 (/opt/ros/foxy/share/rosidl_adapter/cmake)
[0.380s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.424s] -- Found PythonInterp: /usr/bin/python3 (found version "3.8.10") 
[0.588s] CMake Error at /opt/ros/foxy/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:60 (message):
[0.588s]   execute_process(/usr/bin/python3 -m rosidl_adapter --package-name
[0.589s]   ht_nav_variables --arguments-file
[0.589s]   /home/temur/INS-GPS-ROS2/build/ht_nav_variables/rosidl_adapter__arguments__ht_nav_variables.json
[0.589s]   --output-dir
[0.590s]   /home/temur/INS-GPS-ROS2/build/ht_nav_variables/rosidl_adapter/ht_nav_variables
[0.590s]   --output-file
[0.590s]   /home/temur/INS-GPS-ROS2/build/ht_nav_variables/rosidl_adapter/ht_nav_variables.idls)
[0.590s]   returned error code 1:
[0.590s] 
[0.591s]   Error processing 'geometry_msgs/Point.msg gps_pos' of
[0.591s]   'ht_nav_variables/HtNavGpsData': ''Point.msg' is an invalid message name.
[0.591s]   It should have the pattern '^[A-Z][A-Za-z0-9]*$''
[0.591s] 
[0.591s]   Traceback (most recent call last):
[0.592s] 
[0.592s]     File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
[0.592s]       return _run_code(code, main_globals, None,
[0.592s]     File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
[0.593s]       exec(code, run_globals)
[0.593s]     File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/__main__.py", line 19, in <module>
[0.593s]       sys.exit(main())
[0.593s]     File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/main.py", line 53, in main
[0.593s]       abs_idl_file = convert_to_idl(
[0.594s]     File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/__init__.py", line 19, in convert_to_idl
[0.594s]       return convert_msg_to_idl(
[0.594s]     File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/msg/__init__.py", line 28, in convert_msg_to_idl
[0.594s]       msg = parse_message_string(package_name, input_file.stem, content)
[0.594s]     File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/parser.py", line 516, in parse_message_string
[0.594s]       Type(type_string, context_package_name=pkg_name),
[0.594s]     File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/parser.py", line 276, in __init__
[0.594s]       super(Type, self).__init__(
[0.595s]     File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/parser.py", line 204, in __init__
[0.595s]       raise InvalidResourceName(
[0.595s] 
[0.595s]   rosidl_adapter.parser.InvalidResourceName: 'Point.msg' is an invalid
[0.596s]   message name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$'
[0.596s] 
[0.596s] Call Stack (most recent call first):
[0.596s]   /opt/ros/foxy/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[0.596s]   CMakeLists.txt:25 (rosidl_generate_interfaces)
[0.596s] 
[0.597s] 
[0.597s] -- Configuring incomplete, errors occurred!
[0.597s] See also "/home/temur/INS-GPS-ROS2/build/ht_nav_variables/CMakeFiles/CMakeOutput.log".
[0.599s] Invoked command in '/home/temur/INS-GPS-ROS2/build/ht_nav_variables' returned '1': CMAKE_PREFIX_PATH=/opt/ros/foxy /usr/bin/cmake /home/temur/INS-GPS-ROS2/src/ht_nav_variables -DCMAKE_INSTALL_PREFIX=/home/temur/INS-GPS-ROS2/install/ht_nav_variables
